.bidsignore

```diff
@@ -0,0 +1,2 @@
+**/*.asc
```

Comment:
Almost... it needs to be

`**/*.asc.gz`

Reply:
Ah, and one other thing that could be added to the `.bidsignore` file:

`task-*_physio.json`

See https://github.com/bids-standard/bids-validator/issues/1275#issuecomment-827644778

Reply:
ahhhh crap, thanks for catching that

Reply:
```suggestion
**/*.asc.gz
```